src/Providers/Maps/Google/SharedComponents/MapsAndSearchPlaces.ts

```diff
@@ -141,4 +141,61 @@ namespace Provider.Maps.Google.SharedComponents {
 			});
 		});
 	}
+
+	/**
+	 * Class used to manage Google Maps Libraries/Modules
+	 *
+	 * @export
+	 * @class ExternalModules
+	 */
+	export class ExternalModules {
```

Comment:
Why is this class defined in this file? If should be in a different file, as we already know that other modules will required the same approach...

Reply:
![image](https://github.com/user-attachments/assets/64f96519-189a-4bf1-9626-00b41cd488e4)

Reply:
The name of the file probably is not that happy, but the namespace explains it.

Reply:
And by looking into the functions provided here I was thinking this was the right place: 
![image](https://github.com/user-attachments/assets/1868dc45-3d1f-4a72-aca8-305cd5b734bd)

Reply:
It *can* be in the same namespace but definitely *not* in the same file.